### DISCUSSION TITLE
Creating Views

### DISCUSSION THREAD

**Discussion by MentalGear** [supabase/supabase#8789] [Questions] [answered]
I didn't find any UI options to create a view. Is the only option to execute a query in the SQL editor?

**Comment by GaryAustin1** [comment, marked as answer]
Yes you need to create them with SQL.

**Reply by GaryAustin1** [reply]
Yes it will then show up in the dashboard table UI.

**Reply by MentalGear** [reply]
Will it then be available within supabases GUI?
